build/build.xml

```diff
@@ -17,6 +17,7 @@ External Dependencies:
  	<property name="distro.name"				value="commandbox"/>
  	<property name="commandbox.version"			value="3.0.1"/>
  	<property name="commandbox.stableVersion"	value="3.0.0"/>
+ 	<property name="commandbox.stableSHA256"	value="194738f683cbbbaa63f7a2ae5d1fe6c0d09ef8370c4ac0b6c9aa087f6013abc1" />

```

Comment:
See my last reply on the subject. I know that the current stable is set in stone.

We're not going to get around the delay with Homebrew on the BE.  My major concern is not to break existing references to downloads that will cause their master builds to fail and, possibly, cause us to have to re-submit the formula.

> On Feb 21, 2016, at 1:13 AM, Brad Wood <notifications@github.com> wrote:
> 
> In build/build.xml:
> 
> > @@ -17,6 +17,7 @@ External Dependencies:
> >   	<property name="distro.name"				value="commandbox"/>
> >   	<property name="commandbox.version"			value="3.0.1"/>
> >   	<property name="commandbox.stableVersion"	value="3.0.0"/>
> > + 	<property name="commandbox.stableSHA256"	value="194738f683cbbbaa63f7a2ae5d1fe6c0d09ef8370c4ac0b6c9aa087f6013abc1" />
> So to better describe what happens now, the two brew files that are currently generated are in two separate folders. The "stable" file is in the version dir and only changes until we cut that release (or bump the version number past it). After that, the file is "set in stone" and neither the binaries in the folder, nor the hashes ever change again. The BE version of the file is in the commandbox root outside of any of the version folder and always just represents the last build made. To be clear, which file are you looking to keep? I'd like to always have the one in the version folders so people can always go back and install a previous version manually by grabbing the URL to that brew file. If we want a consolidated one, it sounds like the file in the root outside of the version folders will need to be the one that contains both the stable and the BE versions, however I still don't quite get how the BE will never work since there are commits to CommandBox at least once a we ek and I certainly don't want to submit weekly (or daily) pulls to Brew.
> 
> —
> Reply to this email directly or view it on GitHub.
>